DESCRIPTION

```diff
@@ -43,7 +43,8 @@ Suggests:
     rmarkdown,
     ggplot2,
     magrittr (>= 1.5), 
-    ragg
+    ragg,
```

Comment:
should we add the package versions?

Reply:
Sure, see https://r-pkgs.org/description.html#versioning

I am OOO today. Could you add these in? You may want to check the actions install log to see what versions the VMs are using too. 

Thanks!

Reply:
Sure thing, although now that I think about it the package version dependencies might be a larger question... @bpbond do we want to think about a package manager or something? Should we punt this to that decision?

Reply:
I think specifying reasonable minimum versions is almost always a good idea, both for R and the package dependencies. 

>package manager

I would vote punt for now.